### DISCUSSION TITLE
Local M3U Playlist Song Limit

### DISCUSSION THREAD

**Discussion by Lush78** [music-assistant/support#2323] [Q&A] [answered]
I seem to be hitting a 800 song limit on my M3U playlists stored on my SMB NAS.  Is there a limit?  I have a ~6000 song playlist that I would like to play on shuffle throughout my home.

**Comment by OzGav** [comment, marked as answer]
There isn’t a limit per se but you would need a lot of RAM and decent CPU to be able to make that work

**Comment by Lush78** [comment]
Thank you for the answer.  I can get around the limit using the same playlist on youtube music.  I seem to be able to increase the queue by continually adding to the queue to surpass that limit, but that's a workaround that I can't really automate easily.  

I would like to see the feature added to save the queue as a playlist whenever you feel the time is right to implement that.

Keep up the great work! This is an amazing project that I appreciate very much.

**Comment by Lush78** [comment]
What kind of system would I require for a 6000+ song playlist? I'm currently running HA on a Dell optiplex with a i5 and 8gb of RAM running on HA OS. I want to make Music Assistant one of the core components in my automation and will purchased whatever necessary to achieve that goal of a high performing MA integration.  My current setup seems to handle queues with several thousand songs, but the m3u playlist falls to load/recognize any additional songs after 800. Thanks in advance and I look forward to the future of Music Assistant.

**Reply by OzGav** [reply]
I couldn’t say you will just have to try. As for the apparent 800 song limit you will need to open an issue for that so we can look at it further. Any errors in the logs?
